frontend/src/stores/layout.ts

```diff
@@ -61,10 +71,11 @@ export const useLayoutStore = defineStore("layout", {
         confirm: null,
         action: undefined,
         props: null,
+        close: null,
       });
     },
     closeHovers() {
```

Comment:
This PR was based around the assumption that when closeHovers() was called. it was supposed to close all currently open modals and not just the last modal.  Not sure exactly how it works, though everywhere it is actually used the expectation seems to be "close any open modals" and would have potentially unwanted side effects / more extensive changes if this is only supposed to close the latest modal.

Reply:
True, but there was never an actual use-case for more than 1 active modals being used when it was designed, but then again, it would still only close the last one.